.meta-updater/main.mjs

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+
+// temporary - @vivjs/* should follow same version of `@hms-dbmi/viv` after we have successfully tested a release.
+const VIV_VERSION = '0.12.10-alpha.0';
```

Comment:
This will have to be 13, no?

Reply:
There is no release yet. But I think 0.13 will make sense.